test/integration/scripts/create_pkcs_store.sh

```diff
@@ -82,8 +82,8 @@ tpm2_ptool initpin --label=label --sopin=mysopin --userpin=myuserpin --path=$TPM
 # add 2 aes and 2 rsa keys under tokens 1 and 2
 for t in "label" "wrap-sw"; do
 	echo "Adding 2 AES 256 keys under token \"$t\""
-	tpm2_ptool addkey --algorithm=aes256 --label="$t" --userpin=myuserpin --path=$TPM2_PKCS11_STORE
-	tpm2_ptool addkey --algorithm=aes256 --label="$t" --key-label=mykeylabel --userpin=myuserpin --path=$TPM2_PKCS11_STORE
+	tpm2_ptool addkey --algorithm=aes128 --label="$t" --userpin=myuserpin --path=$TPM2_PKCS11_STORE
+	tpm2_ptool addkey --algorithm=aes128 --label="$t" --key-label=mykeylabel --userpin=myuserpin --path=$TPM2_PKCS11_STORE
```

Comment:
The testing framework is designed to run against the simulator, and I want to test that both keysizes work to ensure that all the plumbing isn't doing anything dumb.